spikeinterface/sorters/runsorter.py

```diff
@@ -41,7 +44,7 @@
     """
 
 
-def run_sorter(sorter_name, recording, output_folder=None,
+def run_sorter(sorter_name, recording, gpu_to_use=None, output_folder=None,
```

Comment:
Can you move the new argument before `docker_image`?

Reply:
I am not sure the problem lies here, but we can try. This does work:
``` python

def some_func():
    pass

a = {}
b = {}
some_func(**a, **b)
```

Reply:
Can you run the tests again now?

Reply:
@caniko can you move the position of the argument? :)